Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -323,6 +324,13 @@ def bump_formula_pr
       alias_rename.map! { |a| formula.tap.alias_dir/a }
     end
 
+    ohai "brew update-python-resources #{formula.name}"
+    unless args.dry_run? && !args.write?
```

Comment:
It would be this:

```ruby
if !args.dry_run? || args.dry_run? && args.write?
```

See https://github.com/Homebrew/brew/pull/8059#discussion_r460058090

I, personally, think the `unless` version is more readable (and avoids having both `||` and `&&`) but two people have requested the longer one so I'll change.